BitmarkRegistry/Helpers/CustomUserDisplay.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CustomUserDisplay.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 7/17/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import Foundation
+
+struct CustomUserDisplay {
+  static func accountNumber(_ accountNumber: String?) -> String? {
```

Comment:
It's not a big deal. You can keep this design. But I'd prefer this to be an extension of `BitmarkSDK.AccountNumber`

Reply:
Cause I have another custom display: such as date format is in uppercase form.
so I think it may be more organized to put every custom display in this struct. :) (cause it's kinda related with UI which we show to user rather than our data)